### PR TITLE
fix: read children's groupKey

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#318

### CODE REVIEW

File: test/unit/InfiniteGrid.spec.js [hunk 1]
```diff
@@ -205,7 +205,58 @@ describe("InfiniteGrid Test", function () {
 					expect(el.style.position).to.be.equals("absolute");
 					expect(el.style.left).to.be.ok;
 				});
-			})
+			});
+			it(`should check children that haven't a groupkey.`, async () => {
```
Comment (WoodNeck) [inline test/unit/InfiniteGrid.spec.js]: Please write a more informative title when writing a TC.
In this case, TC title can be like
"should set the same group key for all children when children don't have a group key"

File: test/unit/InfiniteGrid.spec.js [hunk 2]
```diff
@@ -225,10 +225,10 @@ describe("InfiniteGrid Test", function () {
 					expect(item.groupKey).to.be.above(100000000);
 				});
 			});
-			it(`should check children that have a empty string groupkey.`, async () => {
+			it(`should set to groupkey when children have a groupkey.`, async () => {
```
Comment (WoodNeck) [inline test/unit/InfiniteGrid.spec.js]: `set to` should be just `set`.
So, this can be like
"should set a group key when children have it"

Comment (daybrush) [reply test/unit/InfiniteGrid.spec.js]: @WoodNeck 

This means set it to the children's groupkey.

Comment (WoodNeck) [reply test/unit/InfiniteGrid.spec.js]: Isn't children is the collection of HTMLElement?
Also, `set to` has a quite different meaning from `set` as it refers to start something.

**Pull request merged**